projects/packages/my-jetpack/_inc/components/product-interstitial/index.jsx

```diff
@@ -115,10 +115,11 @@ export default function ProductInterstitial( {
 	);
 
 	const trackProductClick = useCallback(
-		( isFreePlan = false, customSlug = null ) => {
```

Comment:
We can as well introduce `options` object here, to make a bit more order.

We could also think about unifying `trackProductClick` and `trackBundleClick` (and if possible maybe moving it into hook to have a cleaner approach rather than passing props?)

Let me know what you think.

Reply:
I like the idea of unifying those two functions, I don't think a hook is necessary in this case since only these interstitials are using it 🤔